ui/ui.go

```diff
@@ -62,10 +62,12 @@ func (h *uiHandler) Route(unsafe bool) http.Handler {
 
 	r := mux.NewRouter()
 	r.HandleFunc("/ui/dashboard", h.Authenticated(h.Dashboard()))
-	r.HandleFunc("/ui/cert", h.Authenticated(h.ListCertificates()))
-	r.HandleFunc("/ui/cert/{id}", h.Authenticated(h.ViewCertificate())).Methods("GET")
-	r.HandleFunc("/ui/cert/{id}/edit", h.Authenticated(h.EditCertificate())).Methods("GET")
-	r.HandleFunc("/ui/cert/{id}/edit", h.Authenticated(h.SaveCertificate())).Methods("POST")
+	r.HandleFunc("/ui/certificates", h.Authenticated(h.ListCertificates()))
+	r.HandleFunc("/ui/certificate/id/{id}", h.Authenticated(h.ViewCertificate())).Methods("GET")
+	r.HandleFunc("/ui/certificate/id/{id}/edit", h.Authenticated(h.EditCertificate())).Methods("GET")
+	r.HandleFunc("/ui/certificate/id/{id}/edit", h.Authenticated(h.SaveCertificate())).Methods("POST")
+
+	r.HandleFunc("/ui/certificate/create", h.Authenticated(h.CreateCertificate()))
```

Comment:
This should probably only be for POST requests

Reply:
Oh I see this URL is also called via POST for form submission, with the creation UI being served via GET. Can we mark this as serving both GET and POST?